ale_linters/rust/cargo.vim

```diff
@@ -70,14 +72,23 @@ function! ale_linters#rust#cargo#GetCommand(buffer, version_output) abort
         let l:default_feature = ''
     endif
 
+    let l:subcommand = l:use_check ? 'check' : 'build'
+    let l:clippy_options = ''
+
+    if ale#Var(a:buffer, 'rust_cargo_use_clippy') && executable('cargo-clippy')
```

Comment:
```suggestion
    if ale#Var(a:buffer, 'rust_cargo_use_clippy')
```

Could you just check if the variable is set here instead? People will have to manually enable `clippy` anyway.